src/librustc_metadata/cstore_impl.rs

```diff
@@ -115,6 +115,8 @@ provide! { <'tcx> tcx, def_id, cdata
     is_foreign_item => { cdata.is_foreign_item(def_id.index) }
     describe_def => { cdata.get_def(def_id.index) }
     def_span => { cdata.get_span(def_id.index, &tcx.sess) }
+    stability => { cdata.get_stability(def_id.index) }
+    deprecation => { cdata.get_deprecation(def_id.index) }
```

Comment:
@nikomatsakis this is another place of divergence with @cramertj. Should we be referencing the methods on `cdata`, or moving them into this implementation?